.circleci/config.yml

```diff
@@ -38,6 +38,9 @@ jobs:
       - ios/install-dependencies:
             bundle-install: true
             pod-install: true
+      - run:
+          name: Install Dependencies
+          command: rake dependencies
```

Comment:
Providing the `mobile-secrets` hash is up-to-date, this should prevent this kind of failure in the future, right?